index.html

```diff
@@ -82,203 +79,204 @@ <h5>Multi-Post Stories</h5>
-          <ul>
-            <li>Database Management</li>
-            <li>Version Control</li>
-            <li>CLI</li>
-            <li>API Design</li>
-            <li>Web Development</li>
-          </ul>
-        </div>
       </div>
-    </section>
-    <!-- contact form -->
-    <section class="contact-form">
-      <p>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </p>
-      <form action="https://formspree.io/f/xbjezgwz" method="POST">
-        <fieldset>
-          <input
-            type="text"
-            name="user_name"
-            placeholder="Full name"
-            maxlength="30"
-            required
-          />
-          <input type="email" name="user_email" placeholder="Email" required />
-          <textarea
-            name="user_message"
-            maxlength="500"
-            placeholder="Write me something..."
-            required
-          ></textarea>
-          <input
-            class="button"
-            type="submit"
-            name="submit"
-            value="Get in touch"
-          />
-        </fieldset>
-      </form>
-      <img src="Images/Rectangle 42.png" alt="footerdevider" />
-      <ul class="social-media">
-        <li class="social_icon">
-          <a href="https://github.com/rabayarabu"
-            ><img src="Images/Icon -GitHub.png" alt="GitHubIcon"
-          /></a>
-        </li>
-        <li class="social_icon">
-          <a href="https://www.linkedin.com/in/rabaya-rabu-142721169/"
-            ><img src="Images/Icon - Linkedin.png" alt="LinkedinIcon"
-          /></a>
-        </li>
-        <li class="social_icon">
-          <a href=""><img src="Images/Icon - Angelist.png" alt="Angelist" /></a>
-        </li>
-        <li class="social_icon">
-          <a href="https://twitter.com/rabaya_rabu"
-            ><img src="Images/Icon - Twitter.png" alt="TwitterIcon"
-          /></a>
-        </li>
-        <li class="social_icon">
-          <a href=""><img src="Images/Icon - Medium.png" alt="MediumIcon" /></a>
-        </li>
-      </ul>
-      <!-- <img class="footer" src="Images/Light.png" alt="footerBar"> -->
-    </section>
-  </body>
+    </div>
+  </section>
+  <!-- contact form -->
+  <section class="contact-form">
+    <p>
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/xbjezgwz" method="POST">
+      <fieldset>
+        <input
+          type="text"
+          name="user_name"
+          placeholder="Full name"s
+          maxlength="30"
+          required
+        />
+        <input type="email" name="user_email" placeholder="Email" required />
+        <textarea
+          name="user_message"
+          maxlength="500"
+          placeholder="Write me something..."
+          required
+        ></textarea>
+        <input
+          class="button"
```

Comment:
- [ ] Please, as it was pointed out earlier, kindly implement the button interactions such as hover, active (when it's pressed), and disabled according to the given template this will make your app to be more interactive. 👍 

<img width="448" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/230015306-20546829-d47d-4923-927b-dd529dcd1a70.png">